packages/compass-sidebar/src/components/multiple-connections/active-connections/active-connection-list.tsx

```diff
@@ -0,0 +1,108 @@
+import { Subtitle, css, spacing } from '@mongodb-js/compass-components';
+import React, { useEffect, useState } from 'react';
+import { useActiveConnections } from '@mongodb-js/compass-connections/provider';
+import { ActiveConnection } from './active-connection';
+import {
+  getConnectionTitle,
+  type ConnectionInfo,
+} from '@mongodb-js/connection-info';
+
+const activeConnectionsContainerStyles = css({
+  height: '100%',
+  padding: spacing[3],
+});
+
+const activeConnectionListHeaderStyles = css({
+  flexGrow: 0,
+  display: 'flex',
+  flexDirection: 'row',
+  alignContent: 'center',
+  justifyContent: 'space-between',
+});
+
+const activeConnectionListHeaderTitleStyles = css({
+  marginTop: 0,
+  marginBottom: 0,
+  textTransform: 'uppercase',
+  fontSize: '12px',
+});
+
+const activeConnectionCountStyles = css({
+  fontWeight: 'normal',
+});
+
+const activeConnectionsListStyles = css({
+  listStyle: 'none',
+  height: '100%',
+});
+
+export function ActiveConnectionList({
+  testUseActiveConnections,
+}: {
+  testUseActiveConnections?: () => ConnectionInfo[];
```

Comment:
I know this is unusual, but I really wanted to avoid pulling too much of the connection logic into the component test.

Reply:
We have a pattern for using `__TEST_<something>` props for passing values to the components that are otherwise impossible to mock, but as you are dealing with services here, the injection mechanism for them is a provider, and I'd suggest to follow this approach and set up providers with mock services that return values you need for that purpose

Reply:
updated the test